e2e/dractions/retry.go

```diff
@@ -46,7 +46,7 @@ func waitPlacementDecision(client client.Client, namespace string, placementName
 
 		if time.Since(startTime) > time.Second*time.Duration(util.Timeout) {
 			return "", fmt.Errorf(
-				"could not get placement decision for " + placementName + " before timeout, fail")
+				"could not get placement decision for %s before timeout, fail", placementName)
```

Comment:
Using %q will add quotes around the value, making it the log more clear, and revealing issues with leading or trailing whitespace in the value.

Reply:
should i change also the ones that already use %s to %q too?

Reply:
> should i change also the ones that already use %s to %q too?

This can be the next PR, lets focus now on fixing the logs that do not use the format properly.

Reply:
done